docs/source/api/index.rst

```diff
@@ -0,0 +1,24 @@
+`tf-encrypted`
+==============
+
+The top tfe module.
+
+.. code-block:: python
+
+    import tensorflow_encrypted as tfe
+
+
+.. automodule:: tensorflow_encrypted
+  :members: set_protocol, set_config
```

Comment:
Do we have to specify every function to be documented?

Reply:
no . this page is not in use currently